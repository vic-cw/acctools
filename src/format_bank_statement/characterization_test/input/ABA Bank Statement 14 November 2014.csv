Account 012345678910 Posted Operations
Operation Date and Time;Operation Amount;Operation Description;Amount in account currency;Debit/Credit Date;Remain amount
15/10/2014;;Opening balance;0.00;15/10/2014;123.45
20/10/2014;25.00 USD;Purchase-POS Purchase-POS/SKYPE COMMUNICATIO, LUXEMBOURG, LU/Orig. auth. amount:USD25/Orig. clear. amount:USD25/Txn. time:15.10.2014 00:00:00/9876 54xx xxxx 3210/TWO ID:19698579/Doc.No:8333687-VISA-19OCT2014;-25.00;20/10/2014;123.45
21/10/2014;5.00 USD;Misc Debit E-SERVICE FROM SIN DOE JOHN;-5.00;21/10/2014;123.45
27/10/2014;25.00 USD;Purchase-POS Purchase-POS/SKYPE COMMUNICATIO, LUXEMBOURG, LU/Orig. auth. amount:USD25/Orig. clear. amount:USD25/Txn. time:21.10.2014 00:00:00/9876 54xx xxxx 3210/TWO ID:19862175/Doc.No:8421009-VISA-26OCT2014;-25.00;27/10/2014;123.45
28/10/2014;25.00 USD;Purchase-POS Purchase-POS/SKYPE COMMUNICATIO, LUXEMBOURG, LU/Orig. auth. amount:USD25/Orig. clear. amount:USD25/Txn. time:23.10.2014 00:00:00/9876 54xx xxxx 3210/TWO ID:19944780/Doc.No:8437289-VISA-27OCT2014;-25.00;28/10/2014;123.45
03/11/2014;25.00 USD;Purchase-POS Purchase-POS/SKYPE COMMUNICATIO, LUXEMBOURG, LU/Orig. auth. amount:USD25/Orig. clear. amount:USD25/Txn. time:29.10.2014 00:00:00/9876 54xx xxxx 3210/TWO ID:20182961/Doc.No:8521038-VISA-02NOV2014;-25.00;03/11/2014;123.45
14/11/2014;;Closing balance;0.00;14/11/2014;123.45
20/11/2014;;Existing blocked amounts;0.00;20/11/2014;123.45
26/03/2013;;Mail/Phone Order, CARD : 9876 **** **** 4321, TRANS. DATE : 26-MAR, TIME : 00:00:00, LOCATION : P13^^ - SKYPE COMMUNICATIO, TERM. : 28021785, TRACE : 41278, TWOID : 5774399, ORIG.AMNT : 25, ORIG.CCY : USD;-25.00;26/03/2013;123.45
20/11/2014;;PAN:9876**..**4321, CARD HOLDER:JOHN DOE, Orig. amount:99.99/USD, Orig. time:20/11/2014 13:17:09, From acct:012345678910 (DOE JOHN), Retailer:000980020296995, Termowner:PAYPAL *LINKEDIN, Terminal:00000001, Country:UNITED KINGDOM, TWOID:20941479;-99.99;20/11/2014;123.45
20/11/2014;;Total blocked amounts;+158.42;20/11/2014;123.45
