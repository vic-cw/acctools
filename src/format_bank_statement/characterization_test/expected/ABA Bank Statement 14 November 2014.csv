20/10/2014,25.00 USD,"Purchase-POS Purchase-POS/SKYPE COMMUNICATIO, LUXEMBOURG, LU/Orig. auth. amount:USD25/Orig. clear. amount:USD25/Txn. time:15.10.2014 00:00:00/9876 54xx xxxx 3210/TWO ID:19698579/Doc.No:8333687-VISA-19OCT2014",-25.00,20/10/2014,123.45
21/10/2014,5.00 USD,Misc Debit E-SERVICE FROM SIN DOE JOHN,-5.00,21/10/2014,123.45
27/10/2014,25.00 USD,"Purchase-POS Purchase-POS/SKYPE COMMUNICATIO, LUXEMBOURG, LU/Orig. auth. amount:USD25/Orig. clear. amount:USD25/Txn. time:21.10.2014 00:00:00/9876 54xx xxxx 3210/TWO ID:19862175/Doc.No:8421009-VISA-26OCT2014",-25.00,27/10/2014,123.45
28/10/2014,25.00 USD,"Purchase-POS Purchase-POS/SKYPE COMMUNICATIO, LUXEMBOURG, LU/Orig. auth. amount:USD25/Orig. clear. amount:USD25/Txn. time:23.10.2014 00:00:00/9876 54xx xxxx 3210/TWO ID:19944780/Doc.No:8437289-VISA-27OCT2014",-25.00,28/10/2014,123.45
03/11/2014,25.00 USD,"Purchase-POS Purchase-POS/SKYPE COMMUNICATIO, LUXEMBOURG, LU/Orig. auth. amount:USD25/Orig. clear. amount:USD25/Txn. time:29.10.2014 00:00:00/9876 54xx xxxx 3210/TWO ID:20182961/Doc.No:8521038-VISA-02NOV2014",-25.00,03/11/2014,123.45
